src/utils/polyfill.ts

```diff
@@ -0,0 +1,13 @@
+declare global {
+  interface ObjectConstructor {
+    keys<O>(object: O): (keyof O)[];
+  }
+
+  interface PromiseConstructor {
+    delay(time: number): Promise<void>;
+  }
+}
+
+Promise.delay = (time) => new Promise((next) => setTimeout(next, time));
```

Comment:
I think it is better to create a `delay` function instead of modify the `Promise` object, it may cause misunderstanding for other people (because they cant search in MDN)

Reply:
Ok! i think so, it not good
I created a new function for this function but not in this PR, because it's a lot of code, I can't pay attention to all
Thank for this.